.github/workflows/ci.yml

```diff
@@ -35,9 +35,13 @@ jobs:
         with:
           version: "${{ matrix.NXF_VER }}"
 
+      - name: Connect to Github Container Registry
+        uses: docker/login-action@v2.2.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
When we were still private this was needed to fetch container images from the Github container registry. This is just to authenticate docker using the default GITHUB_TOKEN so it can pull any private Github container registry image for which we explicitly add access. It is not needed anymore since everything is now public and not hosted on the gh registry anymore. I'll remove it.